mappings/net/minecraft/client/render/VertexConsumer.mapping

```diff
@@ -68,8 +68,8 @@ CLASS net/minecraft/class_4588 net/minecraft/client/render/VertexConsumer
 		ARG 5 green
 		ARG 6 blue
 		ARG 7 alpha
-		ARG 8 u
-		ARG 9 v
+		ARG 8 textureU
+		ARG 9 textureV
```

Comment:
Why change this? By convention, the letters `u` and `v` represent the texture coordinates, the same way `x`, `y`, and `z` represent position coordinates.

Reply:
I felt like `u` and `v` were insufficiently descriptive.

Reply:
`u` and `v` could also refer to lightmap coords.

Reply:
`u` and `v` are by definition **texture coordinates**. Adding the word texture doesn't solve any kind of ambiguity (the lightmap is also a texture).